src/js/vivliostyle-viewer.js

```diff
@@ -18,8 +18,6 @@
  */
 
 import ko from "knockout";
-import menuButtonBinding from "./bindings/menuButton.js";
-import swipePagesBinding from "./bindings/swipePages.js";
```

Comment:
These two imports are necessary to enable menuButton UI effect and swipePages functionalities.
Instead of removing these imports, replacing to the following code will be good:

```js
import "./bindings/menuButton";
import "./bindings/swipePages";
```

Reply:
fixed!